### PR TITLE
add delete function

### PR DESCRIPTION
**Pull request by IrisHerholz**

### CODE REVIEW

File: pages/entryListPage.js [hunk 1]
```diff
@@ -10,34 +10,38 @@ export default function EntryListHome() {
...
+      { id: nanoid(),
         ...newJournalEntry,
-        id: uuidv4(),
       },
       ...oldJournalEntries,
     ]);
   };
 
+  function handleDeleteJournalEntry(id) {
+    setJournalEntries(journalEntries.filter((JournalEntry) => JournalEntry.id !== id)
```
Comment (shebtastic) [inline pages/entryListPage.js]: you should camelcase non-component variables with a starting lowercase

File: pages/entryListPage.js [hunk 2]
```diff
@@ -10,34 +10,38 @@ export default function EntryListHome() {
...
-        })}
+        {journalEntries.map((journalEntry) => (
+          <li key={journalEntry.id}>
+            <p>{journalEntry.destination}</p>
+            <p>
+              from {journalEntry.from} to {journalEntry.to}
+            </p>
+            <p>Titel: {journalEntry.entryTitle}</p>
+            <p>Entry: {journalEntry.entry}</p>
+            <button type="delete" onClick={() => handleDeleteJournalEntry(journalEntry.id)}>
```
Comment (shebtastic) [inline pages/entryListPage.js]: `delete` is not a legal button type. here is a link to [allowed values for type](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type).

File: pages/entryListPage.js [hunk 3]
```diff
@@ -10,34 +10,38 @@ export default function EntryListHome() {
...
-            </li>
-          );
-        })}
+        {journalEntries.map((journalEntry) => (
+          <li key={journalEntry.id}>
+            <p>{journalEntry.destination}</p>
+            <p>
+              from {journalEntry.from} to {journalEntry.to}
+            </p>
+            <p>Titel: {journalEntry.entryTitle}</p>
```
Comment (shebtastic) [inline pages/entryListPage.js]: title sounds like there could be a more semantic tag for this

File: pages/entryListPage.js [hunk 4]
```diff
@@ -10,34 +10,38 @@ export default function EntryListHome() {
...
-              <p>Entry: {journalEntry?.entry}</p>
-            </li>
-          );
-        })}
+        {journalEntries.map((journalEntry) => (
+          <li key={journalEntry.id}>
+            <p>{journalEntry.destination}</p>
+            <p>
+              from {journalEntry.from} to {journalEntry.to}
+            </p>
```
Comment (shebtastic) [inline pages/entryListPage.js]: these tags might not actually be paragraphs

File: pages/entryListPage.js [hunk 5]
```diff
@@ -1,7 +1,7 @@
-import { v4 as uuidv4 } from "uuid";
+
```
Comment (shebtastic) [inline pages/entryListPage.js]: if you are no longer going to use the `uuid` package for generating ids and are switched to `nanoid`, you should make that change consistently across your app and uninstall the package.

**Review by shebtastic — CHANGES_REQUESTED** [review]
While browsing through the code that was not part of the PR, I noticed some inconsistencies:

- [These lines should not do anything.](https://github.com/IrisHerholz/capstone-project/blob/2da496d3e2f67de1363d7a8bb14471f527951006/components/Form/index.js#L30-L31)
- [This state seems to complicate your state handling, but it does not seem to actually add to your functionality.](https://github.com/IrisHerholz/capstone-project/blob/2da496d3e2f67de1363d7a8bb14471f527951006/components/Form/index.js#L12)
- I noticed that you seem to have stopped to use `prettier` to autoformat your code. If you reenable this, writing and reviewing code will be much smoother with fixed indents.  
- Some of your styles don't seem to do anything. Please check your styled for validity and effect.

A lot of the requested changes that are not necessarily part of this PR, but should be changed.

File: components/Form/index.js [hunk 6]
```diff
@@ -1,108 +1,89 @@
...
-  display: grid;
-  top: 50px;
-  border: 2px solid black;
-  border-radius: 2rem;
-  gap: 0,5rem;
-  padding: 10px;
-  margin: 10px;
+const StyledDate = styled.div`
+  display: flex;
+  flex-direction: oooooo;
```
Comment (RobinWitt) [inline components/Form/index.js]: flex-direction should be "column" or "row" (row is the default, so you might not need to put it)

**Review by RobinWitt — APPROVED** [review]
I see you made the suggested changes. Approved with just one small comment.

File: components/Form/index.js (hunk 6 above)
Comment (IrisHerholz) [reply components/Form/index.js]: jap, you are right. i removed it, thx :)

**Pull request merged**